tools/pom.xml

```diff
@@ -2,154 +2,162 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This file looks like it has changed a bit too much.

I believe we use 2 space indent for xml files, and this looks like a change to 4 space indent.